tests/examples/tokens/test_erc20.py

```diff
@@ -0,0 +1,346 @@
+# Author: Takayuki Jimba (@yudetamago), Ryuya Nakamura (@nrryuya)
+# Modified from Philip Daian's tests:
+# https://github.com/ethereum/vyper/blob/v0.1.0-beta.5/tests/examples/tokens/ERC20_solidity_compatible/test/erc20_tests_1.py
+import pytest
+
+from web3.exceptions import (
+    ValidationError
+)
+
+ZERO_ADDRESS = '0x0000000000000000000000000000000000000000'
+MAX_UINT256 = (2 ** 256) - 1  # Max uint256 value
+
+
+@pytest.fixture
+def c(get_contract, w3, bytes_helper):
+    with open('examples/tokens/ERC20.vy') as f:
+        code = f.read()
+    name = bytes_helper("Vypercoin", 32)
+    symbol = bytes_helper("VYP", 32)
+    c = get_contract(code, name, symbol, 0, 0)
+    return c
+
+
+@pytest.fixture
+def c_bad(get_contract, w3, bytes_helper):
+    # Bad contract is used for overflow checks on totalSupply corrupted
+    with open('examples/tokens/ERC20.vy') as f:
+        code = f.read()
+    name = bytes_helper("Vypercoin", 32)
+    symbol = bytes_helper("VYP", 32)
+    bad_code = code.replace("self.total_supply = self.total_supply + _value", "") \
+                .replace("self.total_supply = self.total_supply - _value", "")
+    c = get_contract(bad_code, name, symbol, 0, 0)
+    return c
+
+
+@pytest.fixture
+def get_log_args(get_logs):
```

Comment:
We can reuse this fixture in `test_erc721.py` and `test_company.py` so I'll create a new pr after this gets merged and add this fixture to conftest.py.

Reply:
Just check `conftest.py` I feel like we have something similar already ;)

Reply:
You mean [`get_logs`](https://github.com/ethereum/vyper/blob/v0.1.0-beta.5/tests/conftest.py#L300)? We extended it to `get_log_args` which extracts the `args` attribute from `logs`.